### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by monich**

### CODE REVIEW

File: rpm/libqbinder.spec [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+Summary:    Qt integration for libgbinder
+Version:    1.0.0
+Release:    1
+Group:      System/Libraries
+License:    BSD
+URL:        https://github.com/mer-hybris/libqbinder
+Source0:    %{name}-%{version}.tar.bz2
+
+%define libgbinder_version 1.0.40
+%define libgbinder_version 1.0.39
```
Comment (mlehtima) [inline rpm/libqbinder.spec]: Two different versions defined.

File: rpm/libqbinder.spec [hunk 2]
```diff
@@ -0,0 +1,58 @@
+Name:       libqbinder
+
+Summary:    Qt integration for libgbinder
+Version:    1.0.0
+Release:    1
+Group:      System/Libraries
```
Comment (mlehtima) [inline rpm/libqbinder.spec]: We don't set Group anymore in spec files for Sailfish OS. Remove also from devel package.

**Review by mlehtima — CHANGES_REQUESTED** [review]

File: rpm/libqbinder.spec (hunk 1 above)
Comment (monich) [reply rpm/libqbinder.spec]: Once libgbinder PR is merged, I'll remove `%define libgbinder_version 1.0.39`, it was added to test OBS build.

File: rpm/libqbinder.spec (hunk 2 above)
Comment (monich) [reply rpm/libqbinder.spec]: Ok, it's always been kind of pointless.

File: rpm/libqbinder.spec (hunk 1 above)
Comment (monich) [reply rpm/libqbinder.spec]: Removed.

**Review by mlehtima — APPROVED** [review]

**Pull request merged**